tools/RAiDER/cli/raider.py

```diff
@@ -304,8 +304,8 @@ def main(iargs=None):
             out_filename = w.replace("_ztd", "_std")
             f = f.replace("_ztd", "_std")
         elif los.ray_trace():
-            out_filename = w.replace("_ztd", "_ray")
-            f = f.replace("_ztd", "_ray")
+            out_filename = w.replace("_std", "_ray")
```

Comment:
This shouldn't be needed with the if/elif logic? May have been before.

Reply:
in checkArgs we have: `los="ztd" if (isinstance(los, Zenith) or los is None) else "std",`

So this will always be `_std` when ray tracing and needs to be changed to ray